src/GrpcProtos/capture.proto

```diff
@@ -544,6 +544,15 @@ message ThreadStateSlice {
 
   uint64 duration_ns = 6;
   uint64 end_timestamp_ns = 5;
+
+  uint32 wakeup_tid = 7;
+  uint32 wakeup_pid = 8;
```

Comment:
I think `wakeup_tid/pid` is a bit ambiguous as a "wakeup" involves two threads, a "waker" and a "woken". I would go for `waker_tid`/`waker_pid`.

Reply:
The same applies to other types that use the same names.

Reply:
We had so many back and forths about what to name these variables, and different people have different opinions about this. I think wakeup is good enough. Especially since there's a comment that explains what it is responsible for.

Reply:
Fair enough.